api/src/test/java/org/apache/cloudstack/api/command/user/template/UpdateTemplateCmdTest.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.cloudstack.api.command.user.template;
```

Comment:
```suggestion
// Licensed to the Apache Software Foundation (ASF) under one
// or more contributor license agreements.  See the NOTICE file
// distributed with this work for additional information
// regarding copyright ownership.  The ASF licenses this file
// to you under the Apache License, Version 2.0 (the
// "License"); you may not use this file except in compliance
// with the License.  You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.
package org.apache.cloudstack.api.command.user.template;
```

Reply:
License added in commit [06ef971](https://github.com/apache/cloudstack/pull/8362/commits/06ef97158a0d4e544ba5131072e95404e8dee956)